packages/manager/src/components/AreaChart/utils.ts

```diff
@@ -36,6 +36,33 @@ export const humanizeLargeData = (value: number) => {
   return `${value}`;
 };
 
+export const generate12HourTicks = (data: any[], timezone: string) => {
```

Comment:
any[]??

Reply:
this method is created by Jaalah only, & any is there because AreaChart component does not have any type defined for the data

Reply:
Let's leave a code comment regarding the `any[]` for future removal based on what was said previously